### ISSUE TITLE
openff-evaluator conda installation problems

### ISSUE THREAD

**Issue by barmoral** [openforcefield/openff-evaluator#529]
**Describe the bug**

The installation of openff-evaluator in conda is failing to solve environment. The process is killed and installation does not go through.

**To Reproduce**

In a new/blank conda environment run the command "conda install -c conda-forge openff-evaluator".

**Output**

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: - Killed

**Computing environment (please complete the following information):**
 - Operating system: Linux (Ubuntu 20.04.6 LTS)
 - Output of running `conda list`:

| Name | Version | Build | Channel |
|----------|-----------|---------|------------|
|_libgcc_mutex|             0.1            |     conda_forge  |  conda-forge|
|_openmp_mutex|             4.5       |                2_gnu   | conda-forge|
|ca-certificates     |      2023.7.22  |          hbcca054_0 |   conda-forge|
|certifi                   |2016.9.26       |         py36_0   | conda-forge|
|ld_impl_linux-64  |        2.40       |          h41732ed_0 |   conda-forge|
|libffi                    |3.4.2               | h7f98852_5   | conda-forge|
|libgcc-ng             |    13.2.0            |   h807b86a_2 |   conda-forge|
|libgomp                |   13.2.0           |    h807b86a_2  |  conda-forge|
|libnsl                   | 2.0.0               | h7f98852_0   | conda-forge|
|libsqlite                | 3.43.0             |  h2797004_0 |   conda-forge|
|libstdcxx-ng          |    13.2.0        |       h7e041cc_2 |   conda-forge|
|libzlib                  | 1.2.13            |   hd590300_5   | conda-forge|
|ncurses               |    6.4             |     hcb278e6_0  |  conda-forge|
|openssl                |   1.1.1w        |       hd590300_0 |   conda-forge|
|pip                      | 20.0.2           |        py36_1   | conda-forge|
|python                 |   3.6.15       |   hb7a2778_0_cpython  |  conda-forge|
|python_abi           |     3.6        |             2_cp36m  |  conda-forge|
|readline                 | 8.2          |        h8228510_1   | conda-forge|
|setuptools              |  49.6.0   |        py36h5fab9bb_3  |  conda-forge|
|sqlite                    |3.43.0      |         h2c6b66d_0  |  conda-forge|
|tk                        |8.6.13       |        h2797004_0   | conda-forge|
|wheel                  |   0.36.2   |          pyhd3deb0d_0 |   conda-forge|
|xz                        |5.2.6       |         h166bdaf_0   | conda-forge|

**Additional context**

I tried installing the "yank" package on a new conda environment too (using "conda install yank"), and got the same error as output. These issues might be related.

**Comment by j-wags** [comment]
I'm able to run this pretty quickly with two changes:

* We're switching over to [recommending people use the mamba package manager](https://docs.openforcefield.org/en/latest/install.html#installation) (`conda install -c conda-forge mamba`. Mamba is much faster than conda.
* Using `mamba create` instead of ` mamba install` - `install` tries to update the current environment with minimal changes, but your environment above is using python 3.6 which major packages don't publish for any more, so the environment resolution is very complex. If you do `create`, it solves everything with no constraints. 

Separately, YANK has fallen out of repair and so Evaluator [dropped its YANK interface in July](https://github.com/openforcefield/openff-evaluator/blob/ce446555521e9bfa22a232b10326b49244162998/docs/releasehistory.rst#044---july-24-2023). So you probably want to pin `openff-evaluator=0.4.3` in the mamba create command as well if you're looking to use YANK. As a replacement, Perses is rolling out support for hydration free energies instead, but my understanding is that it's still coming. 

So basically, after you install mamba (the first bullet point), do `mamba create -n evaluator_env -c conda-forge openff-evaluator=0.4.2`. This solves the environment for me in just a few seconds. 

Please let me know if this solves your issue!

**Comment by mattwthompson** [comment]
@barmoral are you still having issues with installation?

**Comment by barmoral** [comment]
Thank you for your help! Yes, I was able to install the old versions succesfully, but have not come around to try if everything works fine. I'll keep you posted if anything else comes up. I appreciate your time and attention!
